### DISCUSSION TITLE
Callback to sanitize allowed attribute values for finer control

### DISCUSSION THREAD

**Discussion by nikhilramabhadra** [mganss/HtmlSanitizer#258] [Ideas]
I want to allow onclick as an attribute. However, I want this as an allowed attribute only if the value of the attribute matches a white list. It would be great if you could provide a callback where I can do this.

**Comment by mganss** [comment]
Check out https://github.com/mganss/HtmlSanitizer/wiki/Hooks, specifically the `RemovingAttribute` event.

**Reply by nikhilramabhadra** [reply]
Thank you. I will have a look.

**Reply by nikhilramabhadra** [reply]
Thank you. It worked. I set Cancel = false to continue to remove the attribute or Cancel = true to keep the attribute.
